### PR TITLE
homepage

### PR DESCRIPTION
**Pull request by saviaga**
Homepage
Bought background image
CSS/SCSS
No Dialog for login/register(althought it has a padding/margin issue)
Logo Added

### CODE REVIEW

File: static/css/homepage.css [hunk 1]
```diff
@@ -0,0 +1,198 @@
+/**** BACKGROUND ****/
```
Comment (dmorina) [inline static/css/homepage.css]: scss please!

File: static/templates/base/index.html [hunk 2]
```diff
@@ -13,6 +13,7 @@
     <link href="{% static 'bower_components/angular-material/angular-material.min.css' %}" rel="stylesheet">
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,700,500' rel='stylesheet' type='text/css'>
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link href='http://fonts.googleapis.com/css?family=Source+Sans+Pro:200,400' rel='stylesheet' type='text/css'>
```
Comment (dmorina) [inline static/templates/base/index.html]: use https to remove warnings

File: static/templates/homepage.html [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+                <div layout="col" layout-align="center start" class="footeralign">
+                    GET STARTED
+                    <br>Find a job
+                    <br>Hire workers
+                </div>
+            </div>
+            <br>
+        </div>
+    </md-content>
+</div>
```
Comment (dmorina) [inline static/templates/homepage.html]: having one class per element is not really good, try to re-use properties as much as possible

**Comment by dmorina** [comment]
resolve conflicts..
a link to the site where you bought the image is necessary, we have to be careful with copyrighted content

File: static/templates/base/index.html [hunk 4]
```diff
@@ -50,12 +53,12 @@ <h1 ng-if="!isLoggedIn" layout-align="left">Next Generation Crowdsourcing</h1>
...
 </div>
 
-<div ng-if="!isLoggedIn" id="forthcontainer" style="background-color: #3F51B5;"><canvas id="fourcanvas" width="1417" height="30"></canvas></div>
+<div ng-if="!isLoggedIn" id="forthcontainer" style="background-color: #3F51B5;"><canvas id="fourcanvas" width="1417" height="30"></canvas></div-->
 
 <div id="main" class="main" layout="row" ng-controller="NavbarController as navCtrl">
     <ng-include ng-if="isLoggedIn" src="'/static/templates/base/left_navigation.html'"></ng-include>
-        <div flex="" role="main" tabindex="-1" layout="column" layout-padding>
-            <md-content class="md-middle-content md-padding" md-scroll-y="" ng-view="" >
+        <div flex="" role="main" tabindex="-1" layout="column" ng-layout-padding=" {'layout-padding': 'isLoggedIn'}">
```
Comment (dmorina) [inline static/templates/base/index.html]: as @nistala said earlier, i dont think that this directive exists

**Comment by shirishgoyal** [comment]
Pull request upgraded and all issues resolved 
MD design and layout improved 

Resolved in https://github.com/crowdresearch/crowdsource-platform/pull/494

**Comment by dmorina** [comment]
included in #494 it has a cleaner implementation

**Pull request closed without merging**